docs/en/ingest-management/fleet/upgrade-elastic-agent.asciidoc

```diff
@@ -130,22 +133,55 @@ subscriptions].
 [[view-upgrade-status]]
 == View upgrade status
 
-. On the **Agents** tab, when you trigger an upgrade, agents that are upgrading
- have the status `Updating` until the upgrade is complete, then the status
- changes back to `Healthy`.
-+
+On the Agents tab, when you trigger an upgrade, agents that are upgrading have the status `Updating` until the upgrade is complete, and then the status changes back to `Healthy`.
+
+Agents on version 8.12 and higher that are currently upgrading additionally show a detailed upgrade status indicator.
+
 [role="screenshot"]
-image::images/upgrade-status.png[Screenshot of Agents tab showing upgrade status]
+image::images/upgrade-states.png[Detailed state of an upgrading agent]
+
+The following table explains the upgrade states in the order that they can occur.
+
+.{agent} upgrade states
+|===
+| State | Description
+
+| Upgrade requested | {agent} has received the upgrade notice from {fleet} and the upgrade will start when the request has been received.
```

Comment:
I'd omit the second clause in the description because it isn't accurate for scheduled upgrades.

```suggestion
| Upgrade requested | {agent} has received the upgrade notice from {fleet}.
```